src/Raven.Server/NotificationCenter/RequestTimeTracker.cs

```diff
@@ -16,6 +17,7 @@ public class RequestTimeTracker : IDisposable
         private readonly string _source;
         private readonly bool _doPerformanceHintIfTooLong;
         private readonly Stopwatch _sw;
+        public bool IsDisposed;
```

Comment:
shouldn't this be private and then called `_isDisposed` ?

Reply:
Fixed